src/protocol/properties.rs

```diff
@@ -235,6 +235,7 @@ impl Default for CompletionOptions {
                 ":".to_string(),
                 "(".to_string(),
                 ",".to_string(),
+                "\"".to_string(),
```

Comment:
I'm curious when this trigger occurs?

Reply:
i.e. when I type `r._measurement == "` then it triggers a completion call

Reply:
ahh ok, so the triggered character could be `\"` in that case?